486_Project/Characters_Class.py

```diff
@@ -0,0 +1,64 @@
+class warrior:
+    """
+    warrior class for player
+    :return: none
+    """
+    def __init__(self):
+
+        self.stats = [7, 5, 5, 7]
+        self.warrior_head = 0
+        self.warrior_chest = 1
+        self.warrior_hands = 1
+        self.warrior_legs = 1
+        self.warrior_armor = self.warrior_head + self.warrior_chest + self.warrior_hands + self.warrior_legs
+        self.warrior_max_hp = self.stats[3] * 4
+        self.warrior_hp = self.warrior_max_hp
+        self.player_menu = '''
+        [1]Attack
+        [2]Defend
+        [3]Skills
+        [4]Items
+        '''
+        self.player_skills = '''
+        [1]Big Swing
+        [2]Double Swing
+        '''
+        self.big_cdt = 4
+        self.big_cd = 0
+        self.double_cdt = 3
+        self.double_cd = 0
```

Comment:
It might be worthwhile to have a simple "Skill" class that details things like their names and cooldowns. This would make it easier to add new skills!